auth/domain/src/main/java/com/gkp/auth/domain/AuthInfo.kt

```diff
@@ -1,5 +1,8 @@
 package com.gkp.auth.domain
 
+import kotlinx.serialization.Serializable
+
+@Serializable
```

Comment:
Strictly speaking, domain should be isolated from any external dependencies like Kotlinx Serialization. It's up to you how strict you want to be with that, but in the original sense, domain is about the what (business logic) while data is about the how (implementation details, like how a string is serialized to JSON). Doing so maximizes the reusability of domain, since it can be used without needing anything else and doesn't make assumptions about the project (like that it uses Kotlinx serialization).

If you want to enforce that, you'll need an `AuthInfo` model in both data and domain where the data layer maps the data model to the domain one (which can then be used across the app)